apps/site/assets/css/_events.scss

```diff
@@ -373,3 +367,14 @@ $mobile-control-height: 73px;
 .c-expandable-block-caret--black {
   @include expandable-block-caret($color: $black);
 }
+
+.m-month-control {
+  color: $brand-primary;
+  display: block;
+  font-size: 18px;
+  margin-bottom: $base-spacing / 2;
+
+  &:hover {
+    text-decoration: underline;
+  }
+}
```

Comment:
This wasn't affected by your changes at all, but I just realized that the font-weight for the month links is different than the mocks.  Not on you to fix, but if you happen to be making other changes - mind adding `font-weight: 500` here?